websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({ clientId: "937015924425367643" });
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			details: "Geziniyor",
+			startTimestamp: Math.floor(Date.now() / 1000),
+		},
+		page = document.location.pathname;
+	if (page == "/") presenceData.details = "Anasayfayı görüntülüyor";
+	if (page == "/yetkililer")
+		presenceData.details = "Yetkililer sayfasını görüntülüyor";
+	if (page == "/profile") {
+		presenceData.details = "Profil görüntüleniyor:";
+		presenceData.state = document.querySelector(".user-info").textContent;
+		presenceData.largeImageKey = (<HTMLImageElement>(
+			document.querySelector("img.avatar")
+		)).src;
+	}
+	if (page == "/share") presenceData.details = "Kod paylaşıyor";
+	if (page == "/tools") presenceData.details = "Araçlar sayfasını görüntülüyor";
+
+	if (page == "/codes") {
+		const URL = new URLSearchParams(document.location.search);
```

Comment:
don't shadow globals like that, also this isn't a URL
```suggestion
		const searchParams = new URLSearchParams(document.location.search);
```